consultation_analyser/consultations/dummy_data.py

```diff
@@ -151,5 +215,27 @@ def create_dummy_consultation_from_yaml(
                         ThemeMappingFactory(
                             answer=answer, theme=theme, execution_run=theme_mapping_run
                         )
+                    logger.info(
+                        f"Added theme mappings for question number {question.number} and question part number {question_part.number}"
+                    )
+                    if include_changes_to_themes:
+                        simulate_user_amending_themes(question_part)
 
     return consultation
+
+
+# Will only be run occasionally to create dummy data
+# Not in prod
```

Comment:
maybe we could throw an error if the env is not `local`?

Reply:
`create_dummy_consultation_from_yaml` already throws an error if we're in prod. I do want to be able to run on dev (no strong feelings on pre-prod).